src/dialog/overflow-menu/overflow-menu.component.ts

```diff
@@ -78,6 +80,11 @@ export class OverflowMenu {
 
 	@Input() placement: "bottom" | "top" = "bottom";
 
+	/**
+	 * Sets the icon of trigger button
+	 */
+	@Input() customIcon: TemplateRef<any>;
```

Comment:
Looks like `customTrigger` might be a more accurate name since users can pass anything in a template.

Reply:
That's right, thanks :)

Reply:
The joys of naming ahaha ... `customTrigger` totally works, sorry for jerking you around with `cutomIcon` 😆